templates/api/api.h.tpl

```diff
@@ -170,11 +172,12 @@ public:
 {{- end }}   {{- /* end range operation param*/}}  
     */
 {{- end }}   {{- /* end if operations description */}}
-    virtual {{qtReturn "" .Return }} {{.Name}}({{qtParams "" .Params}}) = 0;
+    virtual {{qtReturn "" .Return }} {{lower1 .Name}}({{qtParams "" .Params}}) = 0;
```

Comment:
you could use `{{ camel .Name }} which makes camel case with lower 1st. or `{{ qtVar .Name }}`

Reply:
I can change if that is what we want, but camel seems to do more, this one emphasizes that we change only one letter  🤔 
the qtVar is less readable so I would stick with lower1 or camel (if we want say we change more)